### PR TITLE
Add missing reloc definitions for C-SKY, RISCV and AARCH64 ##bin

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**

- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Updated the include files to the described archs

**Test plan**

I will need to find a bin with those reloc types to write a test

**Closing issues**

Related to other elf prs and issues

### CODE REVIEW

File: libr/bin/p/bin_elf.inc [hunk 1]
```diff
@@ -645,7 +646,25 @@ static RBinReloc *reloc_convert(struct Elf_(r_bin_elf_obj_t) *bin, RBinElfReloc
...
+		break;
+	case EM_RISCV:
+		switch (rel->type) {
+		case R_RISCV_NONE: break;
+		case R_RISCV_JUMP_SLOT: ADD(64, 0);
+		case R_RISCV_RELATIVE: ADD(64, B);
+		default: ADD(64, GOT); break; // reg relocations
+		}
+		break;
+	case EM_ARM64:
```
Comment (ret2libc) [inline libr/bin/p/bin_elf.inc:659]: there is already EM_AARCH64 case, so this is dupped. If there are errors there, fix them there.

File: libr/bin/format/elf/glibc_elf.h [hunk 2]
```diff
@@ -198,6 +198,7 @@ typedef struct
 #define EM_RH32		38		/* TRW RH-32 */
 #define EM_RCE		39		/* Motorola RCE */
 #define EM_ARM		40		/* ARM */
+#define EM_ARM64        183             /* AArch64 */
```
Comment (ret2libc) [inline libr/bin/format/elf/glibc_elf.h:201]: no need for this, the architecture is called `EM_AARCH64` AFAIK

File: libr/bin/format/elf/glibc_elf.h [hunk 3]
```diff
@@ -2633,6 +2635,8 @@ enum
 #define R_AARCH64_TLS_DTPREL   1029	/* Module-relative offset, 64 bit.  */
 #define R_AARCH64_TLS_TPREL    1030	/* TP-relative offset, 64 bit.  */
 #define R_AARCH64_TLSDESC      1031	/* TLS Descriptor.  */
+// #define R_AARCH64_TLS_TPREL64           1030
```
Comment (ret2libc) [inline libr/bin/format/elf/glibc_elf.h:2638]: remove commented stuff

File: libr/bin/p/bin_elf.inc [hunk 4]
```diff
@@ -661,8 +680,8 @@ static RBinReloc *reloc_convert(struct Elf_(r_bin_elf_obj_t) *bin, RBinElfReloc
 	default: break;
 	}
 
-	#undef SET
-	#undef ADD
+#	undef SET
```
Comment (ret2libc) [inline libr/bin/p/bin_elf.inc:683]: why?

**Comment by ret2libc** [comment]
> @ret2libc i added the elf-sync, but there must be some review to see if gnu is missing some of the archs (csky for example?)

Missing compared to what? What are the other "sources of truth" you are comparing it with? I see csky definitions are there and I guess we'll notice whatever we need when we need them.

**Comment by trufae** [comment]
I've verified all the archs and everything is in place. so i'm totally fine in just keeping the official glibc's elf.h the only change needed is that EM_HEXAGON is named EM_QDSP6. waiting for travis and this should be ready

**Comment by trufae** [comment]
seems like builds.sr.ht is broken, but unrelated to my changes

**Pull request merged**